main.py

```diff
@@ -201,6 +230,9 @@
         for submission in sub.hot():
             if len(results) >= limit:  # Manually enforce your desired limit here
                 continue
+            if 'imgur.com' in submission.url:
```

Comment:
this is fine because we skipping it anyways
![456b6e6b40d028626d82ad13b22f4f39254448e9](https://github.com/user-attachments/assets/abd2704d-d06f-4c2b-a20b-1d2dd4163aec)